src/acquisition/covidcast/csv_to_database.py

```diff
@@ -161,13 +161,12 @@ def main(
       database,
       make_handlers(args.data_dir, args.specific_issue_date),
       logger)
-    logger.info("Finished inserting database rows", row_count = modified_row_count)
-    # the following print statement serves the same function as the logger.info call above
-    # print('inserted/updated %d rows' % modified_row_count)
+    logger.info("Finished inserting/updating database rows", row_count = modified_row_count)
   finally:
+    database.do_analyze()
```

Comment:
Do we want the database to run analyze when there's been an exception?

Reply:
in short, "yes".  `upload_archive` calls `commit` or `rollback` for each file, so an exception does not indicate that rows were not added (note the double negative there).  thus, running an ANALYZE is still prudent.  ANALYZE is also fast/inexpensive to run, just in case.

relatedly, there is also some older technical debt here that i was kicking further down the road -- the call below does not need the `True` argument (to force a call to `commit()`) and the associated comment about committing is a misnomer.  i left them there for the sake of getting this ANALYZE call into the acquisition process faster.  because of the aforementioned commit/rollback calls in `upload_archive`, the True+comment bellow are irrelevant and can be removed.  i am happy to do that.

Reply:
that helps! no changes needed